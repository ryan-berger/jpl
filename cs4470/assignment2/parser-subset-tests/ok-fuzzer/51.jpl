time print "p"
read image "Q?" to u3L.
show 0
time read image "" to d
show d()
let w9M. = K5z.()
