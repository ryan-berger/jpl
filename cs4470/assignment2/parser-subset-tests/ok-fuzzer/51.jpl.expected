(TimeCmd (PrintCmd "p"))
(ReadImageCmd "Q?" (VarArgument u3L.))
(ShowCmd (IntExpr 0))
(TimeCmd (ReadImageCmd "" (VarArgument d)))
(ShowCmd (CallExpr d))
(StmtCmd (LetStmt (ArgLValue (VarArgument w9M.)) (CallExpr K5z.)))
